backend/src/test/java/gov/cdc/usds/simplereport/api/BaseAuthenticatedFullStackTest.java

```diff
@@ -0,0 +1,103 @@
+package gov.cdc.usds.simplereport.api;
+
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+import gov.cdc.usds.simplereport.config.authorization.DemoAuthenticationConfiguration;
+import gov.cdc.usds.simplereport.config.simplereport.DemoUserConfiguration;
+import gov.cdc.usds.simplereport.idp.repository.DemoOktaRepository;
+import gov.cdc.usds.simplereport.service.AddressValidationService;
+import gov.cdc.usds.simplereport.service.OrganizationInitializingService;
+import gov.cdc.usds.simplereport.test_util.TestUserIdentities;
+import java.util.stream.Collectors;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+/** Base class for non-graphQL full stack tests that require authentication. */
+@Slf4j
+public abstract class BaseAuthenticatedFullStackTest extends BaseFullStackTest {
+
+  protected static final String ACCESS_ERROR = "Unauthorized";
+
+  @Autowired private OrganizationInitializingService organizationInitializingService;
+  @Autowired private DemoOktaRepository oktaRepository;
+  @Autowired private DemoUserConfiguration userConfiguration;
+  @MockBean private AddressValidationService addressValidationService;
+
+  @Getter @Setter private String username = null;
+
+  protected void useOrgUser() {
+    username = TestUserIdentities.STANDARD_USER;
```

Comment:
just to make sure I'm understanding correctly, removing the underscore  throughout was to adhere to the style guide right? https://github.com/CDCgov/prime-simplereport/wiki/SimpleReport-Style-Guide#variable-names 🧐

Reply:
Yep! Exactly. Might as well fix the naming while touching the file anyways :)

Reply:
I see! Thank you!! 🎉